test/jobs/sidekiq_monitoring/refresh_status_job_test.rb

```diff
@@ -10,12 +10,15 @@ class RefreshStatusJobTest < ActiveJob::TestCase
       Timecop.return
     end
 
-    test 'job' do
-      connection = Minitest::Mock.new
-      connection.expect(:set, true, ['monitoring:timestamp:sidekiq_performed', '2018-01-22 00:00:00'])
-      Redis.stub(:current, connection) do
+    test 'job sets timestamp in redis' do
+      fakeredis = Redis.new
+      assert_nil fakeredis.get('monitoring:timestamp:sidekiq_performed')
+
+      Sidekiq.stub(:redis, {}, fakeredis) do
```

Comment:
it is strange that this has 3 args. Does this even work?

Reply:
The interface of `Minitest::Mock.stub` has variable length parameters that are passed to the block (in case the stubbed method yields a block)
https://github.com/seattlerb/minitest/blob/master/lib/minitest/mock.rb#LC212